resources/capability_summary.json

```diff
@@ -70,70 +70,49 @@
   "DynamoDB": {
     "administer-resource": [
       "dynamodb:CreateBackup",
-      "dynamodb:CreateGlobalTable",
```

Comment:
These permissions for global tables very likely need to stay. The admin user should be allowed to make replicas.

Reply:
These permissions changes were the result of:
1. updating to the full current list of DDB permissions
2. removing permissions not [supported by DDB resource policy](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/rbac-iam-actions.html)

Here are the `administer-resource` permissions from the full set of DDB perms that the DDB resource policy API rejected:
```
(administer-resource) One or more parameter values were invalid: Invalid policy document: The following action names are invalid: "dynamodb:UpdateGlobalTable", "dynamodb:UpdateGlobalTableSettings", "dynamodb:CreateTable", "dynamodb:UpdateGlobalTableVersion", "dynamodb:CreateGlobalTable", "dynamodb:RestoreTableFromBackup", "dynamodb:RestoreTableFromAwsBackup", "dynamodb:CreateTableReplica", "dynamodb:PurchaseReservedCapacityOfferings", "dynamodb:ImportTable"
```

(I have similar output for `read-config`, `read-data`, and `write-data` if you are interested)

I can see why you logically you want an admin of a table to create a replica. However, I think that probably has to be granted from the Identity side. That aligns more with how `Create[Resource]` is generally not supported in resource policy. And in this specific case, I would think the principal CFn uses to create the table resources would be the one that needs permissions.

wdyt?